### ISSUE TITLE
Use Animated Logo on Header.

### ISSUE THREAD

**Issue by tarunnsingh** [alexa-dev-hub/alexa-community-jaipur#50]
A slight update to replace the current Static Logo of Alexa Dev Hub with the Animated Logo [HERE](https://codepen.io/sophiarobots/pen/LYNwLZm)

**Comment by tarunnsingh** [comment]
@harshalkaigaonkar Would like to make this update?

**Comment by harshalkaigaonkar** [comment]
could you make it labeled as hacktoberfest

**Comment by tarunnsingh** [comment]
**Sure would do that.**
But the **new Hacktoberfest 2020 Rules are as follows**:
```
PRs count if:
Submitted in a repo with the hacktoberfest topic AND
during the month of October AND (
  The PR is merged OR
  The PR is labelled as hacktoberfest-accepted by a maintainer OR
  The PR has been approved
)
```

**Comment by harshalkaigaonkar** [comment]
yeah i'll do that

**Comment by harshalkaigaonkar** [comment]
do you want any changes regarding the position of the gif?

**Comment by tarunnsingh** [comment]
@harshalkaigaonkar also please see the header on mobile devices, actually the logo gets hidden. If the Blogs Button is moved within an expandable menu when on smaller screens, the Main Logo will appear fully.![Screenshot_2020-10-05-13-40-52-774_com.android.chrome.jpg](https://user-images.githubusercontent.com/31896659/95055087-6a58ae80-0710-11eb-87f3-b0ddd2bededc.jpg)


Shall create this as an issue and assign it to you?

**Comment by harshalkaigaonkar** [comment]
could you create a new issue and assign it to me

**Comment by harshalkaigaonkar** [comment]
> do you want any changes regarding the position of the gif?

?

**Comment by tarunnsingh** [comment]
> could you create a new issue and assign it to me

Sure.

**Comment by tarunnsingh** [comment]
> do you want any changes regarding the position of the gif?

**No not as such, it should always be center aligned as it is currently. And the GIF should animate only once when the site loads/reloads.**
Can this be done? @harshalkaigaonkar

**Comment by harshalkaigaonkar** [comment]
> > do you want any changes regarding the position of the gif?
> 
> **No not as such, it should always be center aligned as it is currently. And the GIF should animate only once when the site loads/reloads.**
> Can this be done? @harshalkaigaonkar

yeah!
